t/05-soap-binding.t

```diff
@@ -48,7 +48,7 @@ my $xp = get_xpath($request_xml);
 isa_ok($xp, "XML::LibXML::XPathContext");
 
 my $ua   = LWP::UserAgent->new;
-my $soap = $sp->soap_binding($ua, $slo_url, $idp_cert);
+my $soap = $sp->soap_binding($ua, $slo_url, $idp->cert('signing'));
```

Comment:
Perhaps it would be wise to add two certs for a test so we see a pass.

Reply:
I was thinking about tests that are needed here and you are right for sure